SurveyApp/Modules/Home/Cell/SurveyCollectionCell.swift

```diff
@@ -33,8 +33,7 @@ final class SurveyCollectionCell: UICollectionViewCell {
     func configure(with viewModel: SurveyCollectionCellViewModel) {
         titleLabel.text = viewModel.title
         descriptionLabel.text = viewModel.description
-        guard let imageURL: URL = URL(string: viewModel.coverImageUrl) else { return }
-        imageView.af.setImage(withURL: imageURL)
+        imageView.af.setImage(withURL: viewModel.coverImageUrl!)
```

Comment:
please don't use force unwrap.
```suggestion
        if let coverImageUrl = viewModel.coverImageUrl {
        imageView.af.setImage(withURL: coverImageUrl)
        }
```